src/webauth/index.ts

```diff
@@ -8,24 +7,25 @@ import {
   WebAuthorizeOptions,
   WebAuthorizeParameters,
 } from '../types';
+
 import Auth from '../auth';
-import { Auth0Module } from 'src/internal-types';
 
 /**
  * Helper to perform Auth against Auth0 hosted login page
  *
  * It will use `/authorize` endpoint of the Authorization Server (AS)
  * with Code Grant and Proof Key for Challenge Exchange (PKCE).
  *
- * @export
- * @class WebAuth
  * @see https://auth0.com/docs/api-auth/grant/authorization-code-pkce
  */
 class WebAuth {
-  private domain;
-  private clientId;
-  private agent;
+  private domain: string;
+  private clientId: string;
+  private agent: Agent;
 
+  /**
+   * @ignore
+   */
```

Comment:
Ignored so that the constructor does not appear in the published docs (this is always created internally and we don't export the constructor).